code/style.css

```diff
@@ -2,6 +2,340 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+}
+
+body {
+  background-color: #F9F9F3;
+}
+
+.hero-image {
+  width: 100vw;
+  height: 40vh;
+  background-image: url(./images/motivation.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  display: grid;
+  grid-template-rows: 1rem 0;
+  justify-content: space-evenly;
+}
+
+.inner-hero-header {
+  width: 100vw;
+  height: 7vh;
+  background-color: gray;
+  color: #fff;
+  opacity: .7;
+  margin-top: 0;
+}
+  
+
+.logo-place {
+  padding-top: 5px;
+  padding-left: 10px;
+  z-index: 1;
+  color: whitesmoke;
+
+}
+
+.logo-mobil {
+  font-size: 25px;
+  font-weight: 600;
+  color:white;
+}
+
+.navbar {
+  display: none;
+}
+
+.navbar ul li {
+  list-style: none;
+}
+
+/* Mobil layout -- Signup form*/
+
+.form-wrapper {
+  display: flex; 
+  flex-direction: column;
+  justify-content: top;
+  height: fit-content;
+  border-radius: 15px;
+  background-color: #E2E5DE;
+  padding: 25px;
+  margin-top: 10px;
+  margin-left: 5px;
+  margin-right: 5px;
+  margin-bottom: 10px;
+  padding-left: 20px;
+  padding-right: 20px;
```

Comment:
In the same way that you can use shorthand for the margins in my above comment, you could do the same for padding here. So this could be achieved by writing:
`padding: 0 20px;`
Since you want 0 padding on the top, but 20 px on the left and right.